README.md

```diff
@@ -9,247 +9,128 @@ Also, see the [API documentation](https://aws.github.io/aws-encryption-sdk-c/htm
+The AWS SDK for C++ can be installed with Homebrew, which will install the full AWS SDK. If you 
+only need the AWS SDK for C++ to use the AWS Encryption SDK, you have the option to build only the AWS KMS SDK.
+Follow [these directions](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/setup-linux.html#setup-linux-from-source) 
+and specify the `-DBUILD_ONLY="kms"` flag to `cmake` in the instructions provided.
 
-#### Mac: (Option 1) C and C++ build dependencies
+Now, build and install the AWS Encryption SDK for C:
 
-Both aws-sdk-cpp and aws-c-common are required, but the installation of aws-sdk-cpp will install
-aws-c-common for you.
-
-Do a KMS-only build of the AWS SDK for C++. If you want to use the AWS SDK for C++ for
-other AWS services, you can omit the `-DBUILD_ONLY="kms"` argument, but the build will take much longer.
-
-    git clone -b 1.8.32 https://github.com/aws/aws-sdk-cpp.git
-    mkdir build-aws-sdk-cpp && cd build-aws-sdk-cpp
-    cmake -G Xcode -DBUILD_SHARED_LIBS=ON -DBUILD_ONLY="kms" -DENABLE_UNITY_BUILD=ON ../aws-sdk-cpp 
-    xcodebuild -target install ; cd ..
-
-Now skip to the "Mac: Build and install the AWS Encryption SDK for C" section below.
-
-#### Mac: (Option 2) C only build dependencies
-
-Build and install aws-c-common:
-
-    git clone -b v0.4.42 https://github.com/awslabs/aws-c-common.git
-    mkdir build-aws-c-common && cd build-aws-c-common
-    cmake -G Xcode -DBUILD_SHARED_LIBS=ON ../aws-c-common
-    xcodebuild -target install ; cd ..
-
-#### Mac: Build and install the AWS Encryption SDK for C
-
-Brew installed OpenSSL 1.1 to a place that is not picked up by default so we will
-set the directory manually in our build.
-
-    git clone https://github.com/aws/aws-encryption-sdk-c.git
+    git clone --recurse-submodules https://github.com/aws/aws-encryption-sdk-c.git
     mkdir build-aws-encryption-sdk-c && cd build-aws-encryption-sdk-c
-    cmake -G Xcode -DBUILD_SHARED_LIBS=ON -DOPENSSL_ROOT_DIR="/usr/local/opt/openssl@1.1" ../aws-encryption-sdk-c 
+    cmake -G Xcode -DOPENSSL_ROOT_DIR="/usr/local/opt/openssl@1.1" ../aws-encryption-sdk-c 
     xcodebuild -target install; cd ..
 
-You have successfully built and installed the AWS Encryption SDK for C.
+### Windows Build Recipe
 
-### Building on Windows
+**Note**: See the docker-images folder for some Windows build recipes that automate many of these steps.
 
-**Note**: _see the docker-images folder for some Windows build recipes that automate many of theses steps_
+Start by installing Visual Studio version 15 or later with the Windows Universal C Runtime and [Git for Windows](https://git-scm.com/download/win).
 
-Start by installing Visual Studio version 15 or later with the Windows Universal C Runtime and
-[Git for Windows](https://git-scm.com/download/win).
+Use the "x64 Native Tools Command Prompt" for all commands listed here. Run the following commands in the directory where you want to do the build and installation.
```

Comment:
Why use that command prompt? Is it the only one that works? Or the only one we tested?

Reply:
It's the one to have permissions to install the libraries in the default path and to have all of the developer tools populated on the command line without needing to modify or specify the path

Reply:
Makes sense.